test/maxAge.js

```diff
@@ -82,6 +87,78 @@ describe('Max-Age', () => {
     }
   });
 
+  it('times out a request if cache does not start', async () => {
+    const cache = createCache();
+    sandbox.stub(cache, 'start').callsFake(async () => {
+      await bluebird.delay(10000);
+    });
+
+    const connectionTimeout = 10;
+
+    try {
+      await requestWithCache(cache, { ignoreCacheErrors: false, connectionTimeout });
+    } catch (error) {
+      assert.equal(error.message, 'Starting cache timed out after 10');
```

Comment:
If `requestWithCache` doesn't throw, no assertions will run and so the test may pass incorrectly.